sagemaker-training-compiler/huggingface/tensorflow_multiple_gpu_single_node/language-modeling.ipynb

```diff
@@ -0,0 +1,676 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #8.        float("5e-5") / 48 * hyperparameters["per_device_train_batch_size"]
</pre><p>minimize hard coding and fix the math for updating LR --&gt; </p><pre class="ql-syntax" spellcheck="false">float("5e-5") / native_batch_size * hyperparameters["per_device_train_batch_size"] * num_gpus_per_instance
</pre>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-sagemaker-examples/pull/3049/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='19'/>

Reply:
Done